src/index.js

```diff
@@ -82,7 +83,16 @@ function configureInterceptors(instance, url, { username, password } = {}) {
 
   instance.interceptors.response.use(res => {
     return res.data;
+  }, err => {
+    if (!(err instanceof Error) && typeof err === "object") {
+      return Promise.reject(errorify(err));
+    }
+    return Promise.reject(err);
```

Comment:
Yep that's right.  If it's not an `instanceof Error` then make it a correct error format.  Else, we assume it's correct error format and just `reject` it.  Have a I got the `!` around the wrong way?